docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -2207,7 +2207,8 @@ There are two types of sampling plans:
   * - Race/ethnicity
   * - Tracked Guardian(s) (for noise functions ONLY)
   * - Tracked Guardian Address(es) (for noise functions ONLY)
-  * - Type of GQ (for noise functions ONLY)
+  * - Housing Type (“Household” for an individual household, or one of the six different types of group quarters. Included in ACS, used for noise functions ONLY in CPS)
```

Comment:
I only included the "Housing Type" column in ACS because that's what @aflaxman requested. Is that what we want, or do we want it in CPS as well?

Reply:
I definitely want it in Decennial Census.  To keep things simple, perhaps we should not add it to CPS until we can research it a bit more, as we decided for "relationship to reference person"